src/java/org/tensorics/core/tensor/Positions.java

```diff
@@ -134,12 +145,16 @@ public Position apply(Position position) {
             return Position.of(builder.build());
         }
 
+        public Position from(Position position) {

```

Comment:
I guess this delegation method is just to get a nicer/more fluent syntax elsewhere?

Reply:
Indeed this is the reason ... I thought it is not disturbing too  much ...
you can now write
`Position strippedpos = strip(ACoordinate.class).from(aPosition);`